From d1bf65e2ae4bc48bbbe2e6dbe7ddceef9826f682 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 5 Jul 2019 18:16:51 +0200
Subject: [PATCH 1/1] test/py: not all boards support UEFI runtime reset

As not all boards support resets at runtime do not test for it in the
Python tests.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 test/py/tests/test_efi_selftest.py | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/test/py/tests/test_efi_selftest.py b/test/py/tests/test_efi_selftest.py
index da9e5d6a35..bf49f02ebf 100644
--- a/test/py/tests/test_efi_selftest.py
+++ b/test/py/tests/test_efi_selftest.py
@@ -19,10 +19,6 @@ def test_efi_selftest(u_boot_console):
 	m = u_boot_console.p.expect(['Summary: 0 failures', 'Press any key'])
 	if m != 0:
 		raise Exception('Failures occurred during the EFI selftest')
-	u_boot_console.run_command(cmd='', wait_for_echo=False, wait_for_prompt=False);
-	m = u_boot_console.p.expect(['resetting', 'U-Boot'])
-	if m != 0:
-		raise Exception('Reset failed during the EFI selftest')
 	u_boot_console.restart_uboot();
 
 @pytest.mark.buildconfigspec('cmd_bootefi_selftest')
@@ -62,7 +58,7 @@ def test_efi_selftest_text_input(u_boot_console):
 	"""
 	u_boot_console.run_command(cmd='setenv efi_selftest text input')
 	output = u_boot_console.run_command(cmd='bootefi selftest',
-					    wait_for_prompt=False)
+						wait_for_prompt=False)
 	m = u_boot_console.p.expect(['To terminate type \'x\''])
 	if m != 0:
 		raise Exception('No prompt for \'text input\' test')
@@ -132,7 +128,7 @@ def test_efi_selftest_text_input_ex(u_boot_console):
 	"""
 	u_boot_console.run_command(cmd='setenv efi_selftest extended text input')
 	output = u_boot_console.run_command(cmd='bootefi selftest',
-					    wait_for_prompt=False)
+						wait_for_prompt=False)
 	m = u_boot_console.p.expect(['To terminate type \'CTRL\+x\''])
 	if m != 0:
 		raise Exception('No prompt for \'text input\' test')
-- 
2.20.1

